frontend/packages/knative-plugin/src/yaml-templates.ts

```diff
@@ -13,6 +13,7 @@ spec:
   template:
     spec:
       containers:
-      - image: openshift/hello-openshift
+      - image: registry.redhat.io/rhel8/support-tools
+        command: [ "/bin/bash", "-c", "sleep infinity" ]
```

Comment:
I'm not sure if this will cause problems for any of the Knative walkthroughs. We might want to address this one separarely.

cc @bbrowning

Reply:
I don't think this will actually work for Knative. The image needs to be one that returns a HTTP 200 in response to a HTTP GET at the root path. We are tracking this work on the Serverless side at https://issues.redhat.com/browse/SRVKS-611 but that issue has not been picked up to work on yet. I think perhaps we were missing the fact that this example is hardcoded in the console itself.

@rhuss Do you know of a better supported image we could suggest for the console to use here? Ideally this would reference an image produced by the serverless team, but I don't think we're supporting anything for this use yet and won't be until the above linked JIRA issue is solved.

Reply:
I'm not 100% sure if I grasp the context fully, but for using an image as sample for Knative serving it has to talk HTTP. Ideally such an image would also be able to consume CloudEvents so that it can be used in an Eventing context, too.

@bbrowning I'm not sure if we productise "edisplay" which is used for logging CloudEvents into the Pod's log, but this would be a good candidate IMO. @navidshaikh do you know about any productised image that we could use as an example here ?

Reply:
@rhuss : No, we dont productise such image yet. So we may ship two such images one for hello-example and another for event-display ?

Reply:
I wonder how suitable our test image from https://github.com/openshift/knative-client/tree/master/openshift/ci-operator/knative-test-images for this purpose ? (at least for the smoke test for serving)

Reply:
> So we may ship two such images one for hello-example and another for event-display ?

I thought about shipping a single image that we could use for serving (accepting GET requests) and eventing (accepting POST requests with cloud events as payload) ?